scripts/workset_upload_sql.py

```diff
@@ -23,9 +23,8 @@ def main(args):
         doc={}
         for row in db.view('worksets/lims_id')[ws.obj['id']]:
             doc=db.get(row.id)
-        
-        if doc:
-            final_doc=lutils.merge(ws.obj, doc)
+
```

Comment:
I don't get it, why would this be an issue? The previous code only merged the doc if the doc had content in it rt?

Reply:
That was exactly the problem. When I tried importing a fresh workset, there was no existing doc and therefore it could not be imported.